consensus/crdt/consensus.go

```diff
@@ -276,6 +282,28 @@ func (css *Consensus) Ready(ctx context.Context) <-chan struct{} {
 	return css.readyCh
 }
 
+// IsTrustedPeer returns whether the given peer is taken into account
+// when submitting updates to the consensus state.
```

Comment:
If the given peer is not taken into account when submitting the updates to the state, it doesn't really help by being part of the cluster, does it?
Do you mean to say that the replication factor will achieved by the trusted peers and whatever followers do will be considered as bonus?

Reply:
> If the given peer is not taken into account when submitting the updates to the state, it doesn't really help by being part of the cluster, does it?

It can be any cluster peer that this peer does not trust (like what we call a "follower"). Anyone can broadcast an update, but peers choose which ones to take into account (only those from trusted peers).

> Do you mean to say that the replication factor will achieved by the trusted peers and whatever followers do will be considered as bonus?

No